letsencrypt-auto-source/pieces/bootstrappers/rpm_python3_legacy.sh

```diff
@@ -0,0 +1,73 @@
+# If new packages are installed by BootstrapRpmPython3 below, this version
```

Comment:
I could use a comment, either here or before BootstrapRpmPython3Legacy, explaining what constitutes an "RPM Python 3 Legacy" OS, especially as compared to "RPM Python 3".

Reply:
Comment added.